paparazzi/build.gradle

```diff
@@ -95,6 +95,29 @@ subprojects {
       ])
     }
   }
+
+  // Avoid the native projects in 'libs'
+  plugins.withId('org.jetbrains.kotlin.jvm') {
+    // Create a test task for each supported JDK.
+    [11, 12, 16, 18].each { majorVersion ->
```

Comment:
Should really include 17 as some vendors declared that an LTS release so the probability that someone will have it is high.